standards/code-style.adoc

```diff
@@ -338,12 +342,68 @@ if (condition) {
 }
 ----
 
-This is __bad__:
-
 [source, kotlin]
 ----
 if (condition)
-    body()  // bad!
+    body()  // WRONG!
+----
+
+Braces are not required for `when` branches and `if` statement bodies if they meet BOTH
+the following conditions:
+
+. the conditional and the branch each contain at most a single predicate
+. it fits on a single line
+
+For `if` statements, there is an additional condition that there be no `else if` or `else`
```

Comment:
Do we feel it's clear that this statement is specifically about single line `if` statements? It probably is, but it threw me for a brief loop, so I raise it for consideration.`